host/class/uac/usb_host_uac/examples/audio_player/README.md

```diff
@@ -0,0 +1,12 @@
+| Supported Targets | ESP32-S2 | ESP32-S3 | ESP32-P4 |
+| ----------------- | -------- | -------- | -------- |
+
+# UVC driver example: Video stream
```

Comment:
Outdated description.

It would be nice to provide a simple 'how-to' use this example